h/static/scripts/group-forms/components/forms/TextField.tsx

```diff
@@ -98,7 +98,7 @@ export default function TextField({
   const InputComponent = type === 'input' ? Input : Textarea;
 
   return (
-    <div className="mb-4">
```

Comment:
I removed the implicit margin to let consumers handle spacing via flex gap with no surprises.

We can allow classes to be passed here if needed.

Reply:
Makes sense. The general rule of thumb is that containers should "own" their padding but not margins.